### PR TITLE
update environments for composer.sls

### PR DESCRIPTION
**Pull request by Insolita**
- change env definiton HOME to COMPOSER_HOME, as more appropriate,  
   see https://getcomposer.org/installer source
- add env definition COMPOSER_ALLOW_SUPERUSER for mute warning
  (manageable from pillar)
- add missing env definitions for composer_github_token command

### CODE REVIEW

File: composer.sls [hunk 1]
```diff
@@ -11,12 +11,15 @@ composer_php:
         - unzip
         - git
 
+{% set composer_home =  pillar.get('composer-home', '/root/.config/.composer')%}
```
Comment (cebe) [inline composer.sls]: shouldn't this be `'/root/.config/composer'` instead?

Comment (cebe) [reply composer.sls]: Same in README.

Comment (Insolita) [reply composer.sls]: Whoops, yes

**Review by cebe — APPROVED** [review]

**Comment by cebe** [comment]
Thank you!

**Pull request merged**